Fix newline in chat deletion confirmation box

svojtkova opened the pull request:
### Short description
<!-- Describe this PR in one or two sentences. -->
In the confirmation of deleting a chat message, a new line was not working and it was showing br tag.

### Proposed changes
<!-- Describe this PR in more detail. -->
Added style to p tag for accepting new line
Instead of <br> tag used space unicode character 



### Side effects
<!-- List all related components that have not been explicitly changed but may be affected by this PR -->
Not found any yet



### Resolved issues

Fixes: #1694


__________________________________________________
<!-- Keep this link for the potential reviewer -->
[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

ulliholtgrave commented:
@svojtkova Thank you very much for this PR! :+1: :muscle: I am pretty sure that this fixes the issue, but we have a small problem with our pipeline cache. We fixed it on our develop branch, but I guess you need to rebase your branch to work accordingly. 
You can just follow the steps in our docu: https://digitalfabrik.github.io/integreat-cms/git-flow.html#update-feature-branch :)

charludo reviewed (approved):
Looks good, nice!

timoludwig commented on integreat_cms/cms/templates/generic_confirmation_dialog.html:
```diff
@@ -14,7 +14,7 @@
             <h2 id="confirmation-title"></h2>
             <h3 id="confirmation-subject" class="text-center mt-4 mb-6 break-words"></h3>
-            <p id="confirmation-text" class="mt-4 mb-6">
+            <p id="confirmation-text" class="mt-4 mb-6" style="white-space: pre-line">
```
```suggestion
            <p id="confirmation-text" class="mt-4 mb-6 whitespace-pre-line">
```
I'd prefer tailwind classes over inline css styles.

timoludwig reviewed (changes requested):
Thanks a lot! :+1:

timoludwig reviewed (changes requested):
Thanks, could you squash your commits and rebase your branch to fix the tests?
Also, in the future I guess it could be better to directly create a branch in this repository instead of opening a PR from your fork.

svojtkova commented:
Hello, I squashed commits and rebased it according to the tutorial, but it is still failing the tests. Don't you know what could help? Can it be because of using fork version?

timoludwig commented:
Yes, you need to rebase onto the develop branch of the upstream repository, not of the develop branch of your fork.

svojtkova commented:
I made changes. I hope this time it will pass all the tests.

timoludwig reviewed (approved):
Thanks! :+1:

svenseeberg reviewed (approved):
Looks legitimate. Thanks. Another rebase seems to be needed though.

timoludwig commented on integreat_cms/cms/templates/chat/_chat_messages.html:
```diff
@@ -2,6 +2,7 @@
 {% load static %}
 {% load rules %}
 {% for message in chat_messages %}
+<<<<<<< HEAD
```
Please resolve the git conflicts manually by either taking the `HEAD` version and making your change with replacing the newline tag again or take your `newline in chat deletion confirmation box` state and format the template with djlint.

timoludwig replied:
Ah yeah sorry, you already fixed it :grin:

svojtkova replied:
Yes but I think I need to fix again this generic_confirmation_dialog.html. As I see that generated translation in django is connected to the lines. I try one more time to change it.

timoludwig replied:
You can just run `./dev-tools/djlint.sh` to apply our new HTML formatting style.

timoludwig commented on integreat_cms/cms/templates/generic_confirmation_dialog.html:
```diff
@@ -14,7 +14,12 @@
         <div class="w-full p-4 rounded shadow">
             <h2 id="confirmation-title"></h2>
             <h3 id="confirmation-subject" class="text-center mt-4 mb-6 break-words"></h3>
+<<<<<<< HEAD
             <p id="confirmation-text" class="mt-4 mb-6"></p>
+=======
+            <p id="confirmation-text" class="mt-4 mb-6 whitespace-pre-line">
+            </p>
+>>>>>>> newline in chat deletion confirmation box
```
```suggestion
            <p id="confirmation-text" class="mt-4 mb-6 whitespace-pre-line"></p>
```

timoludwig reviewed (changes requested):
Thanks! :+1:
Now, could you squash your commits one last time? :see_no_evil:

svojtkova commented:
> Thanks! 👍 Now, could you squash your commits one last time? 🙈

Did I squashed it correctly? I am not sure what is wrong with these 3 files I have changed .

timoludwig commented:
> Did I squashed it correctly? I am not sure what is wrong with these 3 files I have changed .

Oh, something went wrong here... maybe one of my commands in the documentation doesn't work for external forks? Maybe you rebased onto the develop branch of your local fork which is a few commits behind? Could you tell me which git commands you executed before force pushing?

In any case, reverting to the previous state with:
```
git reset --hard 1a61278be24a407248cdaf00fee6f77dc59f87b3
git push --force
```
should probably make sense and after that, you can do an interactive rebase again (but without rebasing onto the fork's develop branch, just squashing the three commits should suffice)...

svojtkova commented:
> 
I did the reset. I followed instuctions for Clean up git history and when using command

 `git rebase --interactive $(git merge-base --fork-point origin/develop $(git rev-parse --abbrev-ref HEAD))`

I got list of all previous commits and changed my last 3 commits as squash they grouped together and I pushed it with the previous commits. Probably deleting previous commits and pushing just my squashed commit could work?

timoludwig commented:
> I did the reset. I followed instuctions for Clean up git history and when using command
> 
> `git rebase --interactive $(git merge-base --fork-point origin/develop $(git rev-parse --abbrev-ref HEAD))`

Ok, yeah, this command won't work for forks that are out of sync with the upstream repository.

> I got list of all previous commits and changed my last 3 commits as squash they grouped together and I pushed it with the previous commits. Probably deleting previous commits and pushing just my squashed commit could work?

I would suggest just doing a plain `git rebase --interactive HEAD~3`...

timoludwig reviewed (approved):
Thanks, looks good now! :+1:

svojtkova commented:
Finally passed all the checks. @timoludwig Thanks for help too.
